### ISSUE TITLE
Payara 5.192 sometimes won't start up

### ISSUE THREAD

**Issue by desolatorxxl** [payara/docker-payaraserver-full#91]
Payara won't startup properly in about 2 from 10 `docker run`'s and will shutdown right away, remaining stuck in an unknown state.

It's somewhat easy to reproduce by running `docker run payara/server-full:5.192` a couple of times, till the following error occurs:

```
# docker run payara/server-full:5.192
[Entrypoint] running /opt/payara/scripts/init_1_generate_deploy_commands.sh

[Entrypoint] ignoring /opt/payara/scripts/init.d/*

Executing Payara Server with the following command line:
/docker-java-home/bin/java
-cp
/opt/payara/appserver/glassfish/modules/glassfish.jar
-XX:+UnlockExperimentalVMOptions
-XX:+UnlockDiagnosticVMOptions
-XX:MetaspaceSize=256m
-XX:+UseStringDeduplication
-XX:MaxMetaspaceSize=2g
-XX:+UseCGroupMemoryLimitForHeap
-XX:MaxRAMFraction=1
-XX:+UseG1GC
-XX:MaxGCPauseMillis=500
-Xbootclasspath/p:/opt/payara/appserver/glassfish/lib/grizzly-npn-bootstrap-1.8.jar
-javaagent:/opt/payara/appserver/glassfish/lib/monitor/flashlight-agent.jar
-Djavax.xml.accessExternalSchema=all
-Djava.security.auth.login.config=/opt/payara/appserver/glassfish/domains/production/config/login.conf
-Djavax.net.ssl.trustStore=/opt/payara/appserver/glassfish/domains/production/config/cacerts.jks
-Dorg.glassfish.grizzly.DEFAULT_MEMORY_MANAGER=org.glassfish.grizzly.memory.HeapMemoryManager
-Djdk.tls.rejectClientInitiatedRenegotiation=true
-Djdk.corba.allowOutputStreamSubclass=true
-Dcom.sun.aas.instanceRoot=/opt/payara/appserver/glassfish/domains/production
-Dcom.sun.aas.installRoot=/opt/payara/appserver/glassfish
-Djava.security.policy=/opt/payara/appserver/glassfish/domains/production/config/server.policy
-Dorg.jboss.weld.serialization.beanIdentifierIndexOptimization=false
-Dcom.sun.enterprise.config.config_environment_factory_class=com.sun.enterprise.config.serverbeans.AppserverConfigEnvironmentFactory
-DANTLR_USE_DIRECT_CLASS_LOADING=true
-Djava.awt.headless=true
-Djdbc.drivers=org.apache.derby.jdbc.ClientDriver
-Dorg.glassfish.grizzly.nio.DefaultSelectorHandler.force-selector-spin-detection=true
-Djavax.net.ssl.keyStore=/opt/payara/appserver/glassfish/domains/production/config/keystore.jks
-Djava.library.path=/opt/payara/appserver/glassfish/lib:/usr/java/packages/lib/amd64:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib
com.sun.enterprise.glassfish.bootstrap.ASMain
-prebootcommandfile
/opt/payara/config/pre-boot-commands.asadmin
-upgrade
false
-read-stdin
true
-postbootcommandfile
/opt/payara/config/post-boot-commands.asadmin
-domainname
production
-domaindir
/opt/payara/appserver/glassfish/domains/production
-asadmin-args
--host,,,localhost,,,--port,,,4848,,,--user,,,admin,,,--passwordfile,,,/opt/payara/passwordFile,,,--secure=false,,,--terse=false,,,--echo=false,,,--interactive=false,,,start-domain,,,--verbose=false,,,--watchdog=false,,,--debug=false,,,--domaindir,,,/opt/payara/appserver/glassfish/domains,,,production
-instancename
server
-type
DAS
-verbose
false
-asadmin-classpath
/opt/payara/appserver/glassfish/lib/client/appserver-cli.jar
-debug
false
-asadmin-classname
com.sun.enterprise.admin.cli.AdminMain
-watchdog
false

Launching Payara Server on Felix platform
Jul 04, 2019 7:42:38 AM com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner createBundleProvisioner
INFO: Create bundle provisioner class = class com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.
Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishRuntime@4a5a149c in service registry.
Reading in commandments from /opt/payara/config/pre-boot-commands.asadmin



#!## LogManagerService.postConstruct : rootFolder=/opt/payara/appserver/glassfish
#!## LogManagerService.postConstruct : templateDir=/opt/payara/appserver/glassfish/lib/templates
#!## LogManagerService.postConstruct : src=/opt/payara/appserver/glassfish/lib/templates/logging.properties
#!## LogManagerService.postConstruct : dest=/opt/payara/appserver/glassfish/domains/production/config/logging.properties
[#|2019-07-04T07:42:42.431+0000|INFO|Payara 5.192|javax.enterprise.logging|_ThreadID=19;_ThreadName=RunLevelControllerThread-1562226162259;_TimeMillis=1562226162431;_LevelValue=800;_MessageID=NCLS-LOGGING-00009;|
  Running Payara Version: Payara Server  5.192 #badassfish (build 115)|#]

[#|2019-07-04T07:42:42.440+0000|INFO|Payara 5.192|javax.enterprise.logging|_ThreadID=19;_ThreadName=RunLevelControllerThread-1562226162259;_TimeMillis=1562226162440;_LevelValue=800;_MessageID=NCLS-LOGGING-00010;|
  Server log file is using Formatter class: com.sun.enterprise.server.logging.ODLLogFormatter|#]

[#|2019-07-04T07:42:42.975+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=20;_ThreadName=RunLevelControllerThread-1562226162267;_TimeMillis=1562226162975;_LevelValue=800;_MessageID=NCLS-CORE-00015;|
  Shutdown requested
MultiException stack 1 of 2
com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW]
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:217)
        at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:78)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2248)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.osgi.framework.BundleException: Unable to acquire global lock for resolve. Unresolved requirements: []
        at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
        at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
        at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:209)
        ... 10 more
MultiException stack 2 of 2
java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
        implementation=fish.payara.appserver.micro.services.PayaraInstanceImpl
        name=payara-instance
        contracts={fish.payara.appserver.micro.services.PayaraInstanceImpl,fish.payara.micro.PayaraInstance}
        scope=org.glassfish.hk2.runlevel.RunLevel
        qualifiers={}
        descriptorType=CLASS
        descriptorVisibility=NORMAL
        metadata=runLevelValue={10},Bundle-SymbolicName={fish.payara.micro.payara-micro-service},Bundle-Version={5.192.0}
        rank=0
        loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW],9179084)
        proxiable=null
        proxyForSameScope=null
        analysisName=null
        id=863
        locatorId=0
        identityHashCode=509848599
        reified=false)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2257)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
|#]

[#|2019-07-04T07:42:43.008+0000|SEVERE|Payara 5.192|javax.enterprise.system.core|_ThreadID=20;_ThreadName=RunLevelControllerThread-1562226162267;_TimeMillis=1562226163008;_LevelValue=1000;_MessageID=NCLS-CORE-00016;|
  Startup service failed to start
MultiException stack 1 of 2
com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW]
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:217)
        at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:78)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2248)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.osgi.framework.BundleException: Unable to acquire global lock for resolve. Unresolved requirements: []
        at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
        at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
        at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:209)
        ... 10 more
MultiException stack 2 of 2
java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
        implementation=fish.payara.appserver.micro.services.PayaraInstanceImpl
        name=payara-instance
        contracts={fish.payara.appserver.micro.services.PayaraInstanceImpl,fish.payara.micro.PayaraInstance}
        scope=org.glassfish.hk2.runlevel.RunLevel
        qualifiers={}
        descriptorType=CLASS
        descriptorVisibility=NORMAL
        metadata=runLevelValue={10},Bundle-SymbolicName={fish.payara.micro.payara-micro-service},Bundle-Version={5.192.0}
        rank=0
        loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW],9179084)
        proxiable=null
        proxyForSameScope=null
        analysisName=null
        id=863
        locatorId=0
        identityHashCode=509848599
        reified=false)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2257)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
|#]

[#|2019-07-04T07:42:43.099+0000|WARNING|Payara 5.192|fish.payara.nucleus.notification.NotificationEventBus|_ThreadID=20;_ThreadName=RunLevelControllerThread-1562226162267;_TimeMillis=1562226163099;_LevelValue=900;|
  Tried to unregisterfish.payara.notification.eventbus.core.CDIEventbusNotifierService@266e657b, it may not have been previously registered|#]

[#|2019-07-04T07:42:43.195+0000|INFO|Payara 5.192|org.hibernate.validator.internal.util.Version|_ThreadID=20;_ThreadName=RunLevelControllerThread-1562226162267;_TimeMillis=1562226163195;_LevelValue=800;|
  HV000001: Hibernate Validator 6.0.16.Final|#]

[#|2019-07-04T07:42:43.217+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=21;_ThreadName=RunLevelControllerThread-1562226162275;_TimeMillis=1562226163217;_LevelValue=800;_MessageID=NCLS-CORE-00101;|
  Network Listener http-listener-1 started in: 13ms - bound to [/0.0.0.0:8080]|#]

[#|2019-07-04T07:42:43.255+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=21;_ThreadName=RunLevelControllerThread-1562226162275;_TimeMillis=1562226163255;_LevelValue=800;_MessageID=NCLS-CORE-00101;|
  Network Listener http-listener-2 started in: 5ms - bound to [/0.0.0.0:8181]|#]

[#|2019-07-04T07:42:43.274+0000|WARNING|Payara 5.192|org.glassfish.grizzly.config.GenericGrizzlyListener|_ThreadID=21;_ThreadName=RunLevelControllerThread-1562226162275;_TimeMillis=1562226163274;_LevelValue=900;|
  HTTP/2 (enabled by default) is unsupported with port unification and will be disabled for network listener admin-listener.|#]

[#|2019-07-04T07:42:43.351+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=21;_ThreadName=RunLevelControllerThread-1562226162275;_TimeMillis=1562226163351;_LevelValue=800;_MessageID=NCLS-CORE-00101;|
  Network Listener admin-listener started in: 41ms - bound to [/0.0.0.0:4848]|#]

[#|2019-07-04T07:42:43.352+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=21;_ThreadName=RunLevelControllerThread-1562226162275;_TimeMillis=1562226163352;_LevelValue=800;_MessageID=NCLS-CORE-00087;|
  Grizzly 2.4.3 started in: 775ms - bound to [http-listener-1:8080, http-listener-2:8181, admin-listener:4848]|#]

[#|2019-07-04T07:42:43.458+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=48;_ThreadName=Thread-10;_TimeMillis=1562226163458;_LevelValue=800;_MessageID=NCLS-CORE-00092;|
  Server shutdown initiated|#]

[#|2019-07-04T07:42:43.569+0000|SEVERE|Payara 5.192|javax.enterprise.system.core|_ThreadID=1;_ThreadName=main;_TimeMillis=1562226163569;_LevelValue=1000;_MessageID=NCLS-CORE-00014;|
  Shutdown required
MultiException stack 1 of 1
MultiException stack 1 of 2
com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW]
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:217)
        at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:78)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2248)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.osgi.framework.BundleException: Unable to acquire global lock for resolve. Unresolved requirements: []
        at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
        at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
        at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
        at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:209)
        ... 10 more
MultiException stack 2 of 2
java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
        implementation=fish.payara.appserver.micro.services.PayaraInstanceImpl
        name=payara-instance
        contracts={fish.payara.appserver.micro.services.PayaraInstanceImpl,fish.payara.micro.PayaraInstance}
        scope=org.glassfish.hk2.runlevel.RunLevel
        qualifiers={}
        descriptorType=CLASS
        descriptorVisibility=NORMAL
        metadata=runLevelValue={10},Bundle-SymbolicName={fish.payara.micro.payara-micro-service},Bundle-Version={5.192.0}
        rank=0
        loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [fish.payara.micro.payara-micro-service [323]], State = [NEW],9179084)
        proxiable=null
        proxyForSameScope=null
        analysisName=null
        id=863
        locatorId=0
        identityHashCode=509848599
        reified=false)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2257)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:459)
        at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:478)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.isWouldBlockRightNow(CurrentTaskFuture.java:1194)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.oneJob(CurrentTaskFuture.java:1230)
        at org.glassfish.hk2.runlevel.internal.CurrentTaskFuture$QueueRunner.run(CurrentTaskFuture.java:1168)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
|#]

[#|2019-07-04T07:42:43.574+0000|SEVERE|Payara 5.192|javax.enterprise.system.core|_ThreadID=1;_ThreadName=main;_TimeMillis=1562226163574;_LevelValue=1000;|
  Failed to start, exiting|#]

[#|2019-07-04T07:42:43.577+0000|SEVERE|Payara 5.192||_ThreadID=1;_ThreadName=main;_TimeMillis=1562226163577;_LevelValue=1000;|
  Exception in thread "main" |#]

[#|2019-07-04T07:42:43.578+0000|SEVERE|Payara 5.192||_ThreadID=1;_ThreadName=main;_TimeMillis=1562226163578;_LevelValue=1000;|
  java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishMain.main(GlassFishMain.java:109)
        at com.sun.enterprise.glassfish.bootstrap.ASMain.main(ASMain.java:54)
Caused by: java.lang.RuntimeException: org.glassfish.embeddable.GlassFishException: Server failed to start
        at com.sun.enterprise.v3.server.AppServerStartup.start(AppServerStartup.java:225)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.start(GlassFishImpl.java:83)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishDecorator.start(GlassFishDecorator.java:63)
        at com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishImpl.start(EmbeddedOSGiGlassFishImpl.java:75)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishDecorator.start(GlassFishDecorator.java:63)
        at com.sun.enterprise.glassfish.bootstrap.osgi.OSGiGlassFishImpl.start(OSGiGlassFishImpl.java:71)
        at com.sun.enterprise.glassfish.bootstrap.GlassFishMain$Launcher.launch(GlassFishMain.java:130)
        ... 6 more
Caused by: org.glassfish.embeddable.GlassFishException: Server failed to start
        at com.sun.enterprise.v3.server.AppServerStartup.doStart(AppServerStartup.java:237)
        at com.sun.enterprise.v3.server.AppServerStartup.start(AppServerStartup.java:223)
        ... 12 more
|#]

[#|2019-07-04T07:42:43.579+0000|INFO|Payara 5.192|javax.enterprise.system.core|_ThreadID=49;_ThreadName=Thread-11;_TimeMillis=1562226163579;_LevelValue=800;_MessageID=NCLS-CORE-00092;|
  Server shutdown initiated|#]
```

Observations:
* The container won't shutdown on it's own afterwards
* It doesn't matter if a application gets deployed or not
* The issue doesn't occur in payara/server-full:5.191
* We are also building our own Payara 5.192 image based on alpine and it suffers the same problem

**Comment by rdebusscher** [comment]
Hi,

This behaviour is also reported outside a Docker environment.  We have created an internal ticket for our engineers to look into this issue.

**Comment by desolatorxxl** [comment]
@rdebusscher Thank you for the information! Is there any more public information available where I can follow the progress on this issue?
Sadly the issue is a bit of a showstopper for us right now.

**Comment by rdebusscher** [comment]
@desolatorxxl 
(sorry for the late reply, I was on Holiday)
Sorry, the internal ticket is not accessible for anyone outside Payara. we are currently trying to solve this issue but without a reliable reproducer we can't give any estimates for a solution at the moment.

**Comment by rdebusscher** [comment]
@xrowdominik 

Not sure it is worth adding it to the tag release notes, as only very limited cases are reported and almost everyone can use the version 5.192 without any issue.

**Comment by wlermer** [comment]
Please continue investigation: showstopper here as well. unfortunately not reliably reproducable.

**Comment by deratzmann** [comment]
Hey Rudy.
Do you need more input here? We also face this issue (as described in 4182). Currently the server never starts (we make a heavy use of jms functionality, maybe this causes it?). 
Do you know any workaround for that? Payara 5.192 has also this problem, in fact not that often as 5.193.

Best regards.

**Comment by rdebusscher** [comment]
Hi @deratzmann,

We have found out that this is a race condition in the underlying Felix OSGI framework. We are currently implementing a workaround and testing the solution.

For community users, this fix will be available with 5.194, released at the end of November.

The JMS usage may be contributing, however very unlikely. Since the problem occurs before the server is started and JMS is active. When you have additional libraries installed in the server directories, is probably a contributing factor to the problem.

Regards

**Comment by deratzmann** [comment]
Ok, that sounds promising.
We currently installed libs for using AWS SQS and postgres (postgresql-42.2.5). In fact, that are some more libs.
here are some screen shots:
![image](https://user-images.githubusercontent.com/26092521/65016799-69968400-d925-11e9-9175-d052282cf06e.png)
![image](https://user-images.githubusercontent.com/26092521/65016841-892dac80-d925-11e9-9ce2-b6340e2bf3e5.png)

**Comment by OndroMih** [comment]
This should be resolved in 5.194 within https://github.com/payara/Payara/issues/4182.

**Comment by rdebusscher** [comment]
Fixed in 5.193.1
